src/applications/check-in/components/pages/validate/ValidateDisplay.jsx

```diff
@@ -3,7 +3,7 @@ import { useSelector } from 'react-redux';
 import propTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 import { VaTextInput } from 'web-components/react-bindings';
-import Date from '@department-of-veterans-affairs/component-library/Date';
+import { Date } from '@department-of-veterans-affairs/component-library';
```

Comment:
Without this named import from the root of the package, translation doesn't work and the values don't even get loaded:

![Missing strings in Date react component](https://user-images.githubusercontent.com/2008881/167969998-cc7b691f-0114-4330-94c3-9ef88a3fc74c.png)